features/backend/tax_rates.feature

```diff
@@ -58,6 +59,7 @@ Feature: Tax rates
         Given I am on the tax rate creation page
           And I fill in "Name" with "US Food Tax"
           And I fill in "Amount" with "30"
+          And I fill in "Code" with "TR5"
```

Comment:
It would be best to keep fill in order the same for all scenarios. IMO `code` should be the first, then `name` etc.